Files/Strings/zh-Hant/Resources.resw

```diff
@@ -1770,4 +1770,13 @@
   <data name="UnpinItemFromStart.Text" xml:space="preserve">
     <value>從開始功能表取消釘選</value>
   </data>
+  <data name="ToolTipDescriptionName" xml:space="preserve">
```

Comment:
Strings/Resources files should not be changed.

Reply:
@tsvietOK so, you suggest to remove all references and use static Name, Type and Date?

- ToolTipDescriptionName
- ToolTipDescriptionType
- ToolTipDescriptionDate

Reply:
If you want to add translation, the only Strings/en-US/Resources.resw file should be changed. All other language translation related files will be regenerated automatically. But you don't need to regenerate any translation, just keep Strings/en-US/Resources.resw changed.